### ISSUE TITLE
Importing the numpy c-extensions failed

### ISSUE THREAD

**Issue by Digimartyr** [numpy/numpy#17822]
I'm getting the following error message:

-------------------------------------------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-6-7dd3504c366f> in <module>
----> 1 import pandas as pd

/usr/lib/python3/dist-packages/pandas/__init__.py in <module>
     14 
     15 if missing_dependencies:
---> 16     raise ImportError(
     17         "Unable to import required dependencies:\n" + "\n".join(missing_dependencies)
     18     )

ImportError: Unable to import required dependencies:
numpy: 

IMPORTANT: PLEASE READ THIS FOR ADVICE ON HOW TO SOLVE THIS ISSUE!

Importing the numpy c-extensions failed.
- Try uninstalling and reinstalling numpy.
- If you have already done that, then:
  1. Check that you expected to use Python3.9 from "/usr/bin/python3.9",
     and that you have no directories in your PATH or PYTHONPATH that can
     interfere with the Python and numpy version "1.17.4" you're trying to use.
  2. If (1) looks fine, you can open a new issue at
     https://github.com/numpy/numpy/issues.  Please include details on:
     - how you installed Python
     - how you installed numpy
     - your operating system
     - whether or not you have multiple versions of Python installed
     - if you built from source, your compiler versions and ideally a build log

- If you're working with a numpy git repository, try `git clean -xdf`
  (removes all files not under version control) and rebuild numpy.

Note: this error has many possible causes, so please don't comment on
an existing issue about this - open a new one instead.

Original error was: No module named 'numpy.core._multiarray_umath'
...

All over this website I'm seeing closed topics with the solution to be to simply uninstall then reinstall numpy. This has not worked for me, and I haven't seen any other workable solution.

Is there any other solution other than uninstalling and reinstalling numpy?

I have Python3.9.0
Ubuntu 20.04
Attempting to run pandas through Jupyter Notebook

Thank you.

**Comment by seberg** [comment]
You should investigate why you are using a NumPy version 1.17.4.  That does not really make sense if you are using such a new python version. That indicates something is wrong elsewhere in your setup (e.g. a bad `PYTHONPATH` setup).  https://numpy.org/devdocs/user/troubleshooting-importerror.html might have a tip...

**Comment by Digimartyr** [comment]
I'll give it a shot. I only just downloaded Anaconda. To be honest, having gone through these forums I've uninstalled and installed NumPy multiple times through pip install numpy. So version 1.17.4 would have had to come from pip. I didn't jot down the version that was present when first installing Anaconda, but I believe it was a dated version of numpy.

Otherwise, numpy may be different from NumPy?

**Comment by mattip** [comment]
If you are using anaconda, you should be sure you have activated the environment you installed with `conda activate` and then do `conda install numpy`. Try not to use `pip` in a conda environment.

**Comment by Digimartyr** [comment]
I'm getting the same error message.

**Comment by Digimartyr** [comment]
I'm confused. Python is embedded in my Linux system. I've checked and verified that it's running appropriately. So I doubt that can be the issue. If it is I wouldn't know what exactly I'm doing wrong with Python.

I've also uninstalled and reinstalled Anaconda.

So I'm really at a loss here.

**Comment by charris** [comment]
Ubuntu 20.04 has Python 3.8 as the default. How did you get 3.9?

**Comment by Digimartyr** [comment]
Downloaded it separately. The error message in Jupyter led me to believe that I should be using Python3.9. You know, its been downloaded, but I don't think installed. 

When I type in python --version on my command line it does read 3.8.5. If this is the case then why is pandas requiring that I use 3.9?

**Comment by Digimartyr** [comment]
So, my issue has been solved.

When opening a new Jupyter notebook file, it prompts you to open in a Python3 notebook or Python3.9.0 64 bit.
I had chose Python3.9.0 64 bit because I thought that was what I had. When I think about it although that is currently the new version, and was one I downloaded, it isn't the one that's currently installed on my system. So when opening a Python3 notebook, pandas was able to be imported just fine (no error message about numpy).
